src/Whoops/Util/TemplateHelper.php

```diff
@@ -70,37 +99,56 @@ public function escapeButPreserveUris($raw)
      */
     public function dump($value)
     {
-        if (class_exists('Symfony\Component\VarDumper\Cloner\VarCloner')) {
-            static $dumper = null;
-
-            // re-use the same var-dumper instance, so it won't re-render the global styles/scripts on each dump.
-            if (!$dumper) {
-                $dumper = new HtmlDumper();
-
-                $styles = array(
-                    'default' => '',
-                    'num' => '',
-                    'const' => '',
-                    'str' => '',
-                    'note' => '',
-                    'ref' => '',
-                    'public' => '',
-                    'protected' => '',
-                    'private' => '',
-                    'meta' => '',
-                    'key' => '',
-                    'index' => '',
-                );
-                $dumper->setStyles($styles);
-            }
+        $dumper = $this->getDumper();
 
+        if ($dumper) {
             $cloner = new VarCloner();
-            return $dumper->dump($cloner->cloneVar($value));
+            $output = '';
+            $dumper->dump($cloner->cloneVar($value),  function ($line, $depth) use (&$output) {
```

Comment:
this change seems unrelated, but actually it fixes the `TemplateHelper->dump` method to actually return the dumped html instead of echo'ing it.